tests/logger.go

```diff
@@ -0,0 +1,66 @@
+package tests
+
+import (
+	"io"
+	"sync"
+
+	"github.com/rs/zerolog"
+)
+
+type TestLogger interface {
```

Comment:
we had to implement a custom "log capturer" module as the logger library we're using (zerolog) does not provide one